README.md

```
@@ -21,6 +21,12 @@ Zero‑friction codegen for Drizzle ORM. Analyze your schema. Generate validatio
 
 </div>
 
+## Sponsor
+
+- GitHub Sponsors: https://github.com/sponsors/omar-dulaimi
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)